localization/locales/en.json

```diff
@@ -12,7 +12,13 @@
       "age_limit": "Info: Stale data (min.)",
       "show_age": "Show data age",
       "calc_trend": "Calc trend",
-      "units_in_mmol": "Units in mmol/l"
+      "units_in_mmol": "Units in mmol/l",
+      "always_on_top": "Always on top"
+    },
+    "shortcut": {
+      "title": "Shortcuts",
```

Comment:
The noun `shortcut` typically refers to a link to navigate somewhere. In this context, however, `shortcut` refers to a `hot-key`. Using the name `hot-key` is clearer and more concise in this case.